.ci_support/build_all.py

```diff
@@ -60,6 +60,7 @@ def build_all(recipes_dir, arch):
 
     deployment_version = (0, 0)
     sdk_version = (0, 0)
+    channel_urls = ['local']
```

Comment:
Seems like the changes here are working, I would separate them out into a PR to staged-recipes tomorrow so we don't need to hack anything here.